spec/factories/user.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :user do
+    email "user@email.com"
+    password "password"
+    password_confirmation "password"
+    salt { "asdasdastr4325234324sdfds" }
+    crypted_password { Sorcery::CryptoProviders::BCrypt.encrypt("secret", 
+                     "asdasdastr4325234324sdfds") }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 8, 51 should be on its own line.